.github/workflows/identify-ros-distro.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         id: identify
         run: |
           if ${ROLLING_VAR} == true; then
+            # echo "::set-output name=distro::rolling"
```

Comment:
is it still needed?

Reply:
my bad. was having some trouble with rolling. commented out that line and the did a silly by pasting in the same line. So no, do not need this line.